styles.css

```diff
@@ -158,7 +158,7 @@ button:disabled {
   justify-content: space-between;
 }
 
-.star-button, .x-button {
+.star-button, .x-button, .star-x-button {
```

Comment:
Good call changing this and index.html so our article and button don't have the same class anymore! That could have gotten confusing.